src/translations/main_sv.ts

```diff
@@ -0,0 +1,812 @@
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;När både Thunderbird och Birdtray är inställda på att starta med operativsystemet, resulterar detta oftast i att två exemplar av Thunderbird startas. Här kan du lägga till en fördröjning innan Thunderbird lstartas, för att förhindra detta.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
+    </message>
+    <message>
+        <source> second delay</source>
+        <translation> sekunders fördröjning</translation>
+    </message>
+    <message>
+        <source>with a </source>
+        <translation>med en </translation>
+    </message>
+    <message>
+        <source>and hide it</source>
+        <translation>och dölj den</translation>
+    </message>
+    <message>
+        <source>Terminate Thunderbird when quitting Birdtray</source>
+        <translation>Avsluta Thunderbird när Birdtray avslutas</translation>
+    </message>
+    <message>
+        <source>Hide/show Thunderbird window when clicking on tray icon</source>
+        <translation>Dölj/Visa Thunderbird vid klick på systemfältsikonen</translation>
+    </message>
+    <message>
+        <source>Hide Thunderbird window when it is minimized</source>
+        <translation>Dölj Thunderbird vid minimering</translation>
+    </message>
+    <message>
+        <source>Show red X in tray icon if Thunderbird is not running</source>
+        <translation>Visa ett rött X på systemfältsikonen om Thunderbird inte körs</translation>
+    </message>
+    <message>
+        <source>Restart Thunderbird if it was closed</source>
+        <translation>Starta om Thunderbird om den stängs</translation>
+    </message>
+    <message>
+        <source>New Email</source>
+        <translation>Ny e-post</translation>
+    </message>
+    <message>
+        <source>Enable New Email submenu</source>
+        <translation>Aktivera undermeny till Ny e-post</translation>
+    </message>
+    <message>
+        <source>Advanced</source>
+        <translation>Avancerat</translation>
+    </message>
+    <message>
+        <source>Please do not change these settings unless you understand what you&apos;re doing.</source>
+        <translation>Ändra inte dessa inställningar om du inte säkert vet vad du gör.</translation>
+    </message>
+    <message>
+        <source>Thunderbird window name pattern:</source>
+        <translation>Thunderbirds fönsternamnsmönster:</translation>
+    </message>
+    <message>
+        <source>Minimum notification font size:</source>
+        <translation>Minsta teckenstorlek för avisering:</translation>
+    </message>
+    <message>
+        <source> points</source>
+        <translation> punkter</translation>
+    </message>
+    <message>
+        <source>When blinking, use the fade-in/fade-out transition (increases CPU use!)</source>
+        <translation>Använd in-/uttoning vid blinkning (ökad cpu-belastning)</translation>
```

Comment:
```suggestion
        <translation>Använd in-/uttoning vid blinkning (ökad CPU-belastning)</translation>
```
Is it correct to write the abbreviation in lowercase letters? I'm not sure, if this is correct in Swedish then please ignore this suggestion.